lib/moab/storage_object_validator.rb

```diff
@@ -176,5 +188,21 @@ def check_required_manifest_files(dir, version)
       errors << result if result
       errors
     end
+
+    def druid_from_filepath
+      "druid:#{File.basename(storage_obj_path)}"
+    end
+
+    def latest_druid_version
+      sub_dirs(storage_obj_path).last
```

Comment:
why did you have to add `.last`? could you used `version_directories.last` here?

Reply:
Add if you could use `version_directories.last` then would you need this method ?